### DISCUSSION TITLE
A version switcher for Geometry Dash

### DISCUSSION THREAD

**Discussion by ghost** [geode-sdk/sdk#1165] [Mod Ideas]
A mod idea that could go all the way down to 1.0 even on PC/mobile to experience classic Geometry Dash on that first version what do you guys think? It may require lots of testing but we already have 1.9 out since on Steam that I never saw back in 2.1. You could otherwise make it easier and have just 1.9, 2.0, and 2.1. For a modification of the game.

**Comment by ghost** [comment]
I just realized that this is a stupid idea without RobTop's approval sorry

**Comment by Fleeym** [comment]
At most, this could be done on Windows if you downloaded the versions yourself and then gave the launcher the directory for each version.

We did throw some ideas around about a Geode Launcher for PC but nothing concrete yet.

**Comment by whamer100** [comment]
steam depots could be used, relying on the steam client's console (and also only works if you have a legitimate license of the game on steam)

**Reply by zMidair** [reply]
this is actually genius wtf

**Reply by slideglide** [reply]
or they could rely on DepotDownloader instead of the steam console
